examples/minimal.rs

```diff
@@ -15,9 +15,9 @@ fn main() {
             DefaultPlugins,
             FmodPlugin {
                 audio_banks_paths: &[
-                    "./assets/Master.bank",
-                    "./assets/Master.strings.bank",
-                    "./assets/Music.bank",
+                    "./assets/fmod/demo_project/Build/Desktop/Master.bank",
+                    "./assets/fmod/demo_project/Build/Desktop/Master.strings.bank",
+                    "./assets/fmod/demo_project/Build/Desktop/Music.bank",
```

Comment:
Do we need the /fmod folder? Would look a bit cleaner if the paths are `./assets/fmod_project/Build/Desktop/`

Reply:
Consider this:

My game is called *SuperRacer*.

The project layout looks like this currently:

```
super-racer/
    src/
    assets/
        models/
        textures/
    Cargo.toml
```

Now let's say you want to not name all your FMOD projects "fmod_project", as you may be an audio engineer working on multiple projects.

```
super-racer/
    src/
    assets/
        models/
        super-racer/
            <fmod-files>
        textures/
    Cargo.toml
```

Now, unless you name your FMOD project `fmod` or `fmod_project`, which looks odd in FMOD Studio, you have a random project name again in the `assets/` folder.

That's why I like to nest it in the assets' folder. However, I do think we can make it more generic with `assets/audio/FMOD_PROJECT` instead. What do you think?


```
super-racer/
    src/
    assets/
        audio/
            super-racer/
                <fmod-files>
            raw/
        models/
        textures/
    Cargo.toml
```

In the end, it depends on how the user does it. I just want a somewhat sensible approach, other than putting the FMOD project in the root of the repo.

Reply:
Yeah I was also thinking it's up to the user, so I thought let's keep it as simple as possible in our examples. But either way I'm ok with it.

If we go for the nested folder, `audio` instead of `fmod` is better I agree.

Reply:
Let me change it to `audio` and then I won't touch it again 😅

Reply:
Hahaha sounds good